onedal/cluster/kmeans.py

```diff
@@ -468,17 +509,31 @@ def transform(self, X):
         X_new : ndarray of shape (n_samples, n_clusters)
             X transformed in the new space.
         """
-        check_is_fitted(self)
 
         return self._transform(X)
 
+    def score(self, X, queue=None):
+        """Opposite of the value of X on the K-means objective.
+
+        Parameters
+        ----------
+        X: {array-like, sparse matrix} of shape (n_samples, n_features)
+            New data.
+
+        Returns
+        -------
+        score: float
+            Opposite of the value of X on the K-means objective.
+        """
+        return super()._score(X, self._get_backend("kmeans", "clustering", None), queue)
+
 
 def k_means(
     X,
     n_clusters,
     *,
     init="k-means++",
-    n_init="warn",
+    n_init="auto",
```

Comment:
Any idea why this would've been set to warn originally?

Reply:
At that time this was the default option in sklearn